vmicore/CMakeLists.txt

```diff
@@ -164,6 +192,9 @@ set(test_files
         test/vmi/LibvmiInterface_UnitTest.cpp
         test/vmi/SingleStepSupervisor_UnitTest.cpp)
 
+configure_file(src/config.h.in ${PROJECT_BINARY_DIR}/config.h)
```

Comment:
We already add some definitions via `add_definition()`. It would make sense to consolidate these two methods.